### DISCUSSION TITLE
Help please, Map not loading after update to 2021.02.0

### DISCUSSION THREAD

**Discussion by ps851112** [Hypfer/Valetudo#746] [Q&A / Support] [answered]
Hopefully I didnt over look something during the update.  I just updated to 2021.02.0 from 0.5.1 (Firmware 3.3.0 001886) and map data is not loading.

Persistent Data becomes unchecked after a reboot.
I've tried checking the value and clicking reset map then saving but this doesnt seem to fix my issue.
When clicking save I get a message "The Lab Status is same" from Persistent Data.
[WARN] Error while executing action "enable" for PersistentMapControlCapability { code: -10000, message: 'The Lab Status is same' }

I updated using this URL https://builder.dontvacuum.me/_s5.html w/ and following settings
Uploaded my public key
Replace Xiaomi adbd with generic adbd
Preinstall valetudo newest version
Preinstall Nano texteditor, curl, wget, htop, hexdump
Fixresets patch *new May 2020*
Build for manual installation (requires SSH to install)
S5 (ver 2034, 11/2020, stripped-Ubuntu)
Ran install over ssh from /mnt after extracting the tar file and running the b script then reboot, then a script then reboot.  Everything appears to work aside from the map.

Allowed the vacuum to finish a full cleaning, rebooted, etc.  No map data, does not look like the MQTT data is being sent related to the map either.
Any help is appreciated.

Trace log attached
[valetudo.log](https://github.com/Hypfer/Valetudo/files/6119359/valetudo.log)

**Comment by Hypfer** [comment, marked as answer]
Huh. Weird

There was something sorta-similar on an S4 lately which was solved by changing the `/mnt/data/rockrobo/lab.cfg` to `0` manually via ssh and rebooting. Maybe that helps here as well?

**Reply by ps851112** [reply]
That fixed it, YOU ROCK!!!!  Thank you for all your work on this project!

**Reply by Hypfer** [reply]
I wonder why this is happening.
Maybe there were map data format changes between firmware versions and by skipping some, they don't get migrated properly?
